jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -209,15 +209,15 @@ def run(params) {
             /** Clients stages end **/
 
             /** Products and Salt migration stages begin **/
-            try {
-                stage('Products and Salt migration stages') {
-                    if(params.must_run_products_and_salt_migration_tests){
+            if(params.must_run_products_and_salt_migration_tests) {
+                try {
+                    stage('Products and Salt migration stages') {
                         clientMigrationStages()
-                    }                   
-                } 
-            } catch (Exception ex) {
-                println('ERROR: one or more migrations have failed')
-                products_and_salt_migration_stage_result_fail = true
+                    }
+                } catch (Exception ex) {
+                    println('ERROR: one or more migrations have failed')
```

Comment:
Above in `jenkins_pipelines/environments/common/pipeline-build-validation-aws.groovy` you use a different error message. Is this intentional?